resource/cache/bounded.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/google/webpackager/resource"
+
+	lru "github.com/hashicorp/golang-lru"
+	"golang.org/x/xerrors"
+)
+
+// NewBoundedCache returns a new ResourceCache that stores Resources in memory,
+// with an eviction policy after `size` entries. size must be positive.
+func NewBoundedCache(size int) ResourceCache {
```

Comment:
nit: Should it be NewBoundedInMemoryCache? I don't have a strong preference.

Reply:
Sure, done.